pallets/ajuna-gameregistry/src/lib.rs

```diff
@@ -102,6 +103,7 @@ pub mod pallet {
 	#[pallet::without_storage_info]
 	pub struct Pallet<T>(_);
 
+	// Maybe this pallet is still in an early stage of development, but, why not use Events?
```

Comment:
Yes, this pallet is still actively being developed for L2.
We were notified that Events weren't supported as a callback mechanism from L2 yet. We're relying on storage persistence to communicate between L1 and L2 at the moment as an interim solution.